### DISCUSSION TITLE
Multiple temp and dest drives

### DISCUSSION THREAD

**Discussion by bennyblanco999** [swar/Swar-Chia-Plot-Manager#167] [Q&A]
Hi all - Great work Swar, app works amazing.  I have been doing some testing trying to determine how to maximize my plot speeds but I'm not sure really how to determine that.  

My processor has 32 threads, 16-core.  I have 64GB memory.  2TB Nvme (going up to 3 units) and multiple 16TB drives.

1) Would it be best to RAID0 the 3 x 2TB nVME drives together for speed and simplification?

2) When writing to multiple storage drives (ie. 10 x 16TB) should a new job in the config file be used?  ie. how can I automate filling all the drives in queue?

3) Based on my config above, what would be ideal setup?   With only 2TB nvme, I feel like max concurrent would be 4 (x2 jobs) ?  Does that make sense?

**Comment by bennyblanco999** [comment]
no  one can assist?

**Comment by DANgerous25** [comment]
I also have a similar setup, 32 threads/16 core with 64GB memory and 2TB Nvme (or 1.8 as it really seems to be), but with 7.5 TB drives.

I don't know the answers to your specific questions, as I am still learning the plotting process myself, but I'd love to know. However, for your question 1, I don't think this would be a good idea unless you have a parity drive. So yes, you'd get better write performance but if you lost one drive then you lose all your plots. I don't think the speed of writing to the HDDs is a big issue from what I've learned so far, it won't make much difference to your plotting speed.
